crates/analyzer/src/builtins.rs

```diff
@@ -73,6 +73,8 @@ pub enum Intrinsic {
     __sload,          // (p)
     __sstore,         // (p, v) -> ()
     __msize,          // ()
+    __tstore,         // ()
```

Comment:
the parameter and return count should be added to the functions defined in this module